### ISSUE TITLE
issue about  batch size

### ISSUE THREAD

**Issue by 2133978609** [Wakals/CoVT#25]
Hi, thank you for the great work!

I have a question about the batch size setting used in training. In the paper, the batch size is reported, but it is not entirely clear how this corresponds to the actual training configuration in the code.

Could you please clarify whether the batch size mentioned in the paper refers to the global batch size (i.e., per-device batch size × number of GPUs × gradient accumulation steps), and how BATCH_PER_DEVICE, GLOBAL_BATCH_SIZE, and GPU count were set in your experiments?

**Comment by Wakals** [comment]
Sorry for the confusion. Actually, we have set the global batch size to 4 (if we use one A100, the batch per device is 4; if we use four A6000s, the batch per device is 1); besides, 8 is also good for covt in our experiments (8 A6000, the batch per divice is 1)

Best,

**Comment by 2133978609** [comment]
Thank you for your reply. I have clearly understood your batch size settings, but I am still confused about the definitions of VQA_ONLY_STAGE and MAX_STEPS. I inferred that MAX_STEPS=15000. Which stage does VQA_ONLY_STAGE specifically refer to? I would be very grateful if you could reply

**Comment by 2133978609** [comment]
I have clearly understood the batch size settings, thank you for your reply.
